content/source/layouts/layout.erb

```diff
@@ -69,6 +69,7 @@
               <nav class="navigation-links" role="navigation">
                 <ul class="main-links nav navbar-nav navbar-right">
                   <li><a href="/intro/index.html">Intro</a></li>
+                  <li><a href="https://learn.hashicorp.com/terraform/">Learn</a></li>
```

Comment:
@tr0njavolta It's annoying, but you need to make an additional update in _sidebar.erb... because for some reason there's a separate version of this menu that gets shown as the hamburger menu at mobile widths. (You can see the discrepancy in the local preview if you shrink your browser window narrow enough.)

Reply:
I've updated - thanks!